components/server/src/api/organization-service-api.ts

```diff
@@ -70,9 +76,14 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
         req: UpdateOrganizationRequest,
         context: HandlerContext,
     ): Promise<UpdateOrganizationResponse> {
-        const org = await this.orgService.updateOrganization(context.user.id, req.organizationId, {
-            name: req.name,
-        });
+        if (!uuidValidate(req.organizationId)) {
+            throw new ConnectError("organizationId is required", Code.InvalidArgument);
```

Comment:
`is required` sounds like the parameter is not present, instead of being malformed. Should we have two states?

Reply:
let's keep it simple for now, we will look in some kind of framework for input validations after grpc migration